### PR TITLE
Website changes for MFEM v4.7

### PR DESCRIPTION
**Pull request by tzanio**
### TODO
- [x] Create bit.ly shortlink
- [x] Update version number in `index.md`, `download.md` and `news.md`
- [x] Update list of contributors
- [x] Update publications
- [x] Update the size and SLOC for this version in `download.md`
- [ ] Update doxygen docs
- [ ] Check the links to doxygen, e.g. in `features.md`
- [x] Make sure all new examples are documented

### CODE REVIEW

**Comment by tzanio** [comment]
@victor-decaria-nnl, @arotem3, @j-signorelli, @homeomorfismo and @sethwatts -- we would like to add you to the list of [contributors](https://mfem.org/about/#contributors) as part of the `mfem-4.7` release.

Please let us know if: 1) that's OK with you + 2) is the GitHub page listed in this PR the best URL for you, or do you prefer something else.

Thanks for helping make MFEM better!

**Comment by tzanio** [comment]
We have 4 places where examples and miniapps are listed/documented

:one: https://mfem.org/examples - paragraph, picture and code links in [`examples.md`](https://github.com/mfem/web/blob/master/src/examples.md)

:two: https://mfem.org/features - single line in [`features.md`](https://github.com/mfem/web/blob/master/src/features.md)

:three: https://docs.mfem.org/html/index.html - single line and code links in [`doc/CodeDocumentation.dox`](https://github.com/mfem/mfem/blob/master/doc/CodeDocumentation.dox)

:four: Detailed web pages like https://mfem.org/fluids/ and https://mfem.org/electromagnetics/

Right now, :one:, :two: and :three: above are not in sync and are missing some entries.
Here are a few suggestions to improve that:

- [ ]  @mlstowell - can you add documentation for Example 39 in :one:? (it is already in :two: and :three: in the mfem-4.7 release branches)

- [ ] @jandrej - should we add documentation for Hooke in :one:, :two:, :three: and maybe add a :four: page for it? (see also #175)

- [ ] @ebchin and @psocratis - should we add documentation for the Tribol miniapp in :one:, :two: and :three:? (see also #258)

- [ ] @jandrej - the Multidomain miniapp is in :one:, should we add it to :two: and :three:? 

- [ ] @psocratis - the DPG miniapps are in :three:, should we add them to :one: and :two:?

- [ ] @bslazarov - the AS miniapps are in :three:, should we add them to :one: and :two:? Note that we have https://mfem.org/autodiff.

- [ ] @kmittal2 and @vladotomov - the Mesh Quality miniapp is in :three:, should we add it to :one: and :two:?

_What do you think? Any other miniapp or examples that we should document?_

**Comment by victor-decaria-nnl** [comment]
Thanks @tzanio . That is good with me, and the github page you have for me is fine.

**Review by j-signorelli — APPROVED** [review]
Thank you @tzanio. I am OK with this and the URL for me is best.

**Review by victor-decaria-nnl — APPROVED** [review]

**Review by arotem3 — APPROVED** [review]
Looks good!

File: src/about.md [hunk 1]
```diff
@@ -91,13 +92,16 @@ Please cite with:
...
+- [Amit Rotem](https://github.com/arotem3)
 - [Michael Schneier](https://github.com/nnpp-schneier)
 - [Joachim Schöberl](https://www.asc.tuwien.ac.at/~schoeberl/wiki/index.php/Joachim_Sch%C3%B6berl)
 - [Jean Sexton](https://ccse.lbl.gov/people/jmsexton/index.html)
 - [Syun'ichi Shiraiwa](https://scholar.google.com/citations?user=17PZFkoAAAAJ&hl=en)
 - [Morteza Siboni](https://www.scorec.rpi.edu/~hakimm2)
+- [Joseph Signorelli](https://github.com/j-signorelli)
 - [Cameron Smith](https://www.scorec.rpi.edu/~cwsmith)
 - [Vanessa Sochat](https://vsoch.github.io)
+- [Gabriel Soto](https://github.com/homeomorfismo)
```
Comment (homeomorfismo) [inline src/about.md:104]:
```suggestion
- [Gabriel Pinochet-Soto](https://web.pdx.edu/~gpin2/)
```
Thank you!

**Comment by bslazarov** [comment]
They are good for now - we will introduce changes once the new interface is ready.

**Review by bslazarov — APPROVED** [review]

**Review by mlstowell — APPROVED** [review]

**Pull request merged**